### DISCUSSION TITLE
Is it possible to run the Apple Silicon version of macOS 11 on iPad without architecture emulation?

### DISCUSSION THREAD

**Discussion by UInt2048** [utmapp/UTM#2444] [UTM Help (iOS)] [answered]
Since the Developer Transition Kit contained a A12Z processor, doesn't that mean macOS 11 could run on, say, an iPad Pro - perhaps the fourth-generation one with A12Z, or the fifth-generation rumored to be announced on Tuesday with a A14X that's supposed to be on par with M1?

(This does acknowledge that it might be slow due to lack of hardware acceleration, that it's still being virtualized, etc.)

**Comment by osy** [comment, marked as answer]
The DTK famously lacked virtualization and therefore didn’t support Hypervisor.framework

**Reply by UInt2048** [reply]
@osy You probably heard/saw the announcement of the 5th-gen iPad Pro, which comes with the M1 chip. Does this change anything? (As you know, M1 Macs do support Hypervisor.framework.)

**Reply by osy** [reply]
Apple did not announce hypervisor.framework for iOS so nothing changed.

**Reply by LeonSolisPedro** [reply]
If the DTK lacked virtualization, then... How did Apple demoed a [Linux VM running on a DTK back on June 2020](https://youtu.be/GEZhD3J89ZE?t=6114)? strange....

they probably had a A14 prototype mac around that time...

**Reply by UInt2048** [reply]
@LeonSolisPedro I’m afraid osy is correct; the framework still would have to be ported to iOS in order to be useful to UTM
